src/components/Header.js

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import React, { useState } from 'react';
 import styled from 'styled-components';
 
-export default function Header() {
-	return <Wrapper>발신자 변경해주는 상단바</Wrapper>;
+export default function Header(props) {
+	// 프로필 누구인지
+	const [person, setPerson] = useState(true);
+	
+	const inputTagStyle = {		// 메시지 입력창
+		display: 'flex',
+		height: '100%',
+		width: '70px',
+		borderRadius: '30%',
+	}
+	
+	const showWhoActiveTextStyle = {	// "OO 현재 활동중"
+		displya: 'flex',
+		paddingLeft: '15px',
+		alingContent: 'center',
+
+		paddingTop: '5px'	// 글자 가운데 정렬이 안되어서 임시방편
+	}
+
+	const activeTextStyle = {
+		display: 'flex',
+	}
+	
+	// ChattingScreen컴포넌트에 현재 프로필 전달
+	const formSubmit = e => {
+		e.preventDefault();
+		props.onSubmit(person);
+	}
+
+	// 프로필 클릭 시 변경
+	const onChangePerson = e => {
```

Comment:
주석으로 설명하는 것보다 이렇게 이벤트를 구체적으로 함수명에 담아주는게 더 좋을 것 같아용
```suggestion
	const handleProfileClick = e => {
```